core/modules/comment/comment.install

```diff
@@ -470,6 +470,35 @@ function comment_update_1007() {
   }
 }
 
+/**
+ * New setting for comment titles. Replacing comment_subject_field
+ * with comment_title_options.
+ */
+function comment_update_1008() {
+  foreach (config_get_names_with_prefix('node.type.') as $config_name) {
```

Comment:
As @stpaultim mentioned in the last comment on the issue, we'll need to add a check before this for the contrib module, and convert the current value from the contrib module (if Backdrop) or from the variables table (if D7), and then disable the module.

Reply:
DONE?

Reply:
@stpaultim it's missing a $config-save() but otherwise this hook looks good.

Reply:
@herbdool - Does this look right now?

Reply:
Good